stock_lot_auto_remove/models/stock_warehouse.py

```diff
@@ -0,0 +1,159 @@
+# Copyright 2025 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class StockWarehouse(models.Model):
+
+    _inherit = "stock.warehouse"
+
+    lar_enabled = fields.Boolean(
```

Comment:
IMHO, using abbreviations should be avoided as explicit is better. `lot_autoremove_enabled`